eclass/kernel-install.eclass

```diff
@@ -543,16 +546,18 @@ kernel-install_pkg_pretend() {
 
 	if ! use initramfs && ! has_version "${CATEGORY}/${PN}[-initramfs]"; then
 		ewarn
-		ewarn "WARNING: The standard configuration of the Gentoo distribution"
-		ewarn "kernels requires an initramfs! You have disabled the initramfs"
-		ewarn "USE flag and as a result dracut was not pulled in as a dependency."
-		ewarn "Please ensure that you are either overriding the standard"
-		ewarn "configuration or that an alternative initramfs generation plugin"
-		ewarn "is installed for your installkernel implementation!"
+		ewarn "WARNING: The standard Gentoo distribution kernel configuration is designed"
+		ewarn "to be used with an initramfs! Although possible, there is no guarantee"
+		ewarn "distribution kernels will boot without an initramfs."
 		ewarn
-		ewarn "This is an advanced use case, you are on your own to ensure"
-		ewarn "that your system is bootable!"
+		ewarn "The initramfs USE flag is disabled, and sys-kernel/installkernel is not"
+		ewarn "configured to use an initrd_generator, so an initramfs will not be built."
+		ewarn "Please ensure that an initramfs generator is installed, and installkernel"
+		ewarn "is configured to use it."
 		ewarn
+		ewarn "If a custom initramfs generator is used, installkernel must be"
+		ewarn "configured to use it. Please refer to the installkernel documentation:"
+		ewarn "https://wiki.gentoo.org/wiki/Installkernel#initrd_generator"
 	fi
```

Comment:
This part still needs some attention, it should be more clear that we only modify portage's behaviour and that we don't know what installkernel is going to do so it's user responsibility to ensure it works.

```
		ewarn
		ewarn "WARNING: The default distribution kernel configuration is designed"
		ewarn "to be used with an initramfs! Although possible, there is no guarantee"
		ewarn "that distribution kernels will boot without an initramfs."
		ewarn
		ewarn "You have disabled the initramfs USE flag, and as a result the package manager
        ewarn "will not enforce the configuration of an initramfs generator in"
        ewarn "sys-kernel/installkernel."
        ewarn
		ewarn "If you wish to use a custom initramfs generator, then please ensure that" 
        ewarn "/sbin/installkernel is capable of calling it via a kernel installation hook,"
        ewarn "and is also configured to use it via /etc/kernel/install.conf."
        ewarn
        ewarn "If you wish to boot without an initramfs, then please ensure that"
        ewarn "all kernel drivers required to boot your system are built into the"
        ewarn "kernel by modifying the default distribution kernel configuration"
        ewarn "using /etc/kernel/config.d"
        ewarn
		ewarn "Please refer to the installkernel and distribution kernel documentation:"
		ewarn "    https://wiki.gentoo.org/wiki/Installkernel"
        ewarn "    https://wiki.gentoo.org/wiki/Project:Distribution_Kernel"
```